### PR TITLE
JAVA_OPT and ENV vars setted by setenv.sh template

### PR DESCRIPTION
**Pull request by DuffMck**
---
name: Pull request
about: Describe the proposed change

---

**Describe the change**
A clear and concise description of what the pull request is.

**Testing**
In case a feature was added, how were tests performed?

### CODE REVIEW

File: tasks/instance.yml [hunk 1]
```diff
@@ -36,13 +36,21 @@
...
 
-- name: configure tomcat instance
+- name: configure tomcat instance - server.xml
   template:
     src: server.xml.j2
     dest: "{{ instance.directory
           | default(tomcat_directory) }}/{{ tomcat_archive[instance.version
           | default(tomcat_version)]['dir'] }}/conf/server.xml"
 
+- name: configure tomcat instance - setenv.sh
```
[the remark this answers is not in the document]
Comment (DuffMck) [reply tasks/instance.yml]: Changed!

File: tasks/instance.yml [hunk 2]
```diff
@@ -36,13 +36,21 @@
...
     dest: "{{ instance.directory
           | default(tomcat_directory) }}/{{ tomcat_archive[instance.version
           | default(tomcat_version)]['dir'] }}/conf/server.xml"
 
+- name: configure tomcat instance - setenv.sh 
+  template:
+    src: setenv.sh.j2
+    dest: "{{ instance.directory
+          | default(tomcat_directory) }}/{{ tomcat_archive[instance.version
+          | default(tomcat_version)]['dir'] }}/bin/setenv.sh"
```
[the remark this answers is not in the document]
Comment (DuffMck) [reply tasks/instance.yml]: ok!

File: templates/setenv.sh.j2 [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{% for java_opt in instance.java_opts %}
```
[the remark this answers is not in the document]
Comment (DuffMck) [reply templates/setenv.sh.j2]: This file is read when you start tomcat by command `/bin/sh catalina.sh start`

If you are on windows this file must be a .bat 

Here some [infos](https://docs.oracle.com/cd/E40518_01/integrator.311/integrator_install/src/cli_ldi_server_config.html) from Oracle

File: tasks/instance.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (DuffMck) [reply tasks/instance.yml]: Ok, would you like that I remove this code?

**Comment by robertdebock** [comment]
I've included nearly all changes into the [3.1.0](https://github.com/robertdebock/ansible-role-tomcat/tree/3.1.0) release, which is now being tested and will be released a little later.

With your permission, I'd like to close this pull request and thank you very much for your feedback and cooperation.

**Pull request closed without merging**